### PR TITLE
Update README.

### PR DESCRIPTION
**Pull request by siarheishut**
Add description of Rectangle and Section types.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -120,17 +139,33 @@ Methods:
...
 |-------------------|----------------|-----------------------|
 | SetStrokeColor    | svg::Point     | Sets stroke color.    |
 | SetFillColor      | svg::Point     | Sets fill color.      |
 | SetStrokeWidth    | float          | Sets stroke width.    |
 | SetStrokeLinecap  | std::string    | Sets stroke linecap.  |
 | SetStrokeLineJoin | std::string    | Sets stroke linejoin. |
 
+### svg::Section
+
+Section is an object type with unique properties. It doesn't have most properties, listed above and doesn't specify
```
Comment (foxfrikses) [inline README.md:152]: It doesn't have most properties listed above as it's not a figure.   
A Section is a set of objects, such as Text, Circle, Section etc.

File: README.md [hunk 2]
```diff
@@ -120,17 +139,33 @@ Methods:
...
+any figure itself, instead section may be considered as the set of different objects, such as Circle, Text etc.
+Section has the only one method `Render` which renders all contained figures in their order.
```
Comment (foxfrikses) [inline README.md:154]: Section has the only one method `Render`, which renders all contained objects in the order they were added.

File: README.md [hunk 3]
```diff
@@ -120,17 +139,33 @@ Methods:
...
+
+How to use svg::Section:
+
+1. Create an object of type svg::SectionBuilder.
+2. Use the method `Add` of the builder to add all figures in a set.
+3. Use method `Build` of builder to obtain a `svg::Section` object.
+
+The main purpose of creating Section type is the possibility of copying a set of objects without copying objects
```
Comment (foxfrikses) [inline README.md:162]: The purpose of Section is chip copying of a set of objects without copying the objects themselves.

File: README.md (hunk 1 above)
Comment (siarheishut) [reply README.md:152]: Done.

File: README.md (hunk 2 above)
Comment (siarheishut) [reply README.md:154]: Done.

File: README.md (hunk 3 above)
Comment (siarheishut) [reply README.md:162]: Done.

**Review by foxfrikses — APPROVED** [review]
LGTM

**Pull request merged**